ipn/ipnlocal/local.go

```diff
@@ -222,6 +222,8 @@ type LocalBackend struct {
 	// tkaSyncLock MUST be taken before mu (or inversely, mu must not be held
 	// at the moment that tkaSyncLock is taken).
 	tkaSyncLock sync.Mutex
+
+	capTailnetLockAlpha bool // whether netmap contains the tailnet lock alpha capability
```

Comment:
what are the locking rules for this? It's in a weird section (under a different mutex)

Reply:
looks like `b.mu`, not `tkaSyncLock`.

Reply:
Moved it up to the one for `b.mu`, which is the same for `capFileSharing`.